src/transformers/training_args.py

```diff
@@ -646,6 +648,10 @@ class TrainingArguments:
     label_smoothing_factor: float = field(
         default=0.0, metadata={"help": "The label smoothing epsilon to apply (zero means no label smoothing)."}
     )
+    optim: str = field(
+        default="adamw_hf",
+        metadata={"help": "The optimizer to use: adamw_hf, adamw_torch, adafactor, or apex_fused_adam."},
```

Comment:
```suggestion
        metadata={"help": "The optimizer to use: adamw_hf, adamw_torch, apex_fused_adam or adafactor"},
```

suggesting to regroup `adamw`s together.

additionally, why are we not using the same naming convention for apex?

`adamw_fused_apex` perhaps?